src/test/daily_mode.test.ts

```diff
@@ -28,5 +34,37 @@ describe("Daily Mode", () => {
       expect(pkm.level).toBe(20);
       expect(pkm.moveset.length).toBeGreaterThan(0);
     });
+    expect(game.scene.getModifiers(MapModifier).length).toBeGreaterThan(0);
+  });
+});
+
+/*
+// Need to figure out how to properly start a battle
+// Need to fix eviolite - test keeps insisting it is not in loot table, even though Mini Black Hole (which is using the exact same condition) is
+describe("Shop modifications", async () => {
+  let phaserGame: Phaser.Game;
+  let game: GameManager;
+
+  beforeAll(() => {
+    phaserGame = new Phaser.Game({
+      type: Phaser.HEADLESS,
+    });
+  });
+  beforeEach(() => {
+    game = new GameManager(phaserGame);
+
+    game.override
+      .battleType("single")
+      .startingLevel(200)
+      .moveset([Moves.SURF])
+      .enemyAbility(Abilities.BALL_FETCH);
+    itemPoolChecks.set("EVIOLITE", false);
+    itemPoolChecks.set("MINI_BLACK_HOLE", false);
+  });
+
+  afterEach(() => {
+    game.phaseInterceptor.restoreOg();
+    itemPoolChecks.clear();
   });
 });
+//*/
```

Comment:
Jut add a `.skip` to the describe if you don't want to run it (yet)

```suggestion
// Need to figure out how to properly start a battle
// Need to fix eviolite - test keeps insisting it is not in loot table, even though Mini Black Hole (which is using the exact same condition) is
describe.skip("Shop modifications", async () => {
  let phaserGame: Phaser.Game;
  let game: GameManager;

  beforeAll(() => {
    phaserGame = new Phaser.Game({
      type: Phaser.HEADLESS,
    });
  });
  beforeEach(() => {
    game = new GameManager(phaserGame);

    game.override
      .battleType("single")
      .startingLevel(200)
      .moveset([Moves.SURF])
      .enemyAbility(Abilities.BALL_FETCH);
    itemPoolChecks.set("EVIOLITE", false);
    itemPoolChecks.set("MINI_BLACK_HOLE", false);
  });

  afterEach(() => {
    game.phaseInterceptor.restoreOg();
    itemPoolChecks.clear();
  });
});
```